SuperBuild/External_JsonCpp.cmake

```diff
@@ -0,0 +1,79 @@
+
+set( proj JsonCpp )
+
+# Set dependency list
+set(${proj}_DEPENDENCIES "")
+
+# Include dependent projects if any
+ExternalProject_Include_Dependencies(${proj} PROJECT_VAR proj DEPENDS_VAR ${proj}_DEPENDENCIES)
+
+# Sanity checks
+if(${CMAKE_PROJECT_NAME}_USE_SYSTEM_${proj})
+  message(FATAL_ERROR "Enabling ${CMAKE_PROJECT_NAME}_USE_SYSTEM_${proj} is not supported !")
```

Comment:
Note for later: After contributing support for `JsonCppConfig.cmake` upstream, we should revisit